Statut.md

```diff
@@ -0,0 +1,169 @@
+	8) Organizowanie konkursów, przyznawanie nagród i stypendiów w zakresie obejmującym cele Fundacji;
+	9) Organizowanie akcji medialnych (kampanii informacyjnych i akcji społecznych) promujących działania lub cele Fundacji;
+	10) Upowszechnianie w Internecie, w środkach masowego przekazu, w formie audycji radiowych lub telewizyjnych, filmów telewizyjnych, artykułów prasowych oraz reportaży, wywiadów w prasie, radio i telewizji, i za pomocą wydawnictw wiedzy w zakresie objętym celami Fundacji;
+	11) Współpracę z innymi podmiotami prowadzącymi działalność wspomagającą lub obejmującą cele Fundacji.
+
+§11 Obszary działalności Fundacji
+Fundacja realizuje swoje cele w szczególności w obszarach:
+    1) działalności wspomagającej rozwój gospodarczy, w tym rozwój przedsiębiorczości;
+    2) działalności wspomagającej rozwój techniki, wynalazczości i innowacyjności oraz rozpowszechnianie i wdrażanie nowych rozwiązań technicznych w praktyce gospodarczej;
+    3) promocji i organizacji wolontariatu;oraz działalności charytatywnej;
+    4) działalności wspomagającej rozwój wspólnot i społeczności lokalnych;
+    5) działalności na rzecz organizacji pozarządowych
+
+§12 Współpraca w realizacji celów
+Dla wspierania realizacji swoich celów statutowych Fundacja może przystępować do związków stowarzyszeń oraz innych organizacji społecznych, zawiązywać inne fundacje, których cele będą zbieżne z celami Fundacji, samodzielnie lub przy udziale innych osób fizycznych, osób prawnych oraz jednostek organizacyjnych nieposiadających osobowości prawnej.
+
+Rozdział III: Majątek i dochody Fundacji
+
+§13 Fundusz założycielski i przepływy majątkowe #DoDyskusji
+1. Majątek Fundacji stanowi Fundusz założycielski w kwocie 1000 zł (słownie: tysiąc złotych) oraz środki finansowe, nieruchomości i ruchomości nabyte przez Fundację w toku jej działania.
+2. Środki na realizację celów Fundacji i pokrycie kosztów jej działalności pochodzą z:
+	a) funduszu założycielskiego i darowizn Fundatorów,
+	b) darowizn, spadków i zapisów,
+	c) dotacji, grantów i subwencji osób prawnych,
+	d) dochodów z darów, ze zbiórek i imprez publicznych,
+	e) aukcji internetowych,
+	f) dochodów z majątku nieruchomego i ruchomego,
+	h) odsetek i depozytów bankowych,
+	i) z działalności odpłatnej pożytku publicznego.
+	j) * ... działalności gospodarczej ??? *
+3. Dochody pochodzące z subwencji, darowizn, spadków i zapisów mogą być użyte na wybrane przez Fundację, dowolne działanie związane z realizacją celów Fundacji, o ile ofiarodawcy nie postanowili inaczej.
+4. Fundacja może tworzyć fundusze, w tym o charakterze celowym, zgodnie z wolą ofiarodawcy sygnowane jego nazwiskiem lub nazwą.
+5. Przyjęcie przez Fundację darowizny lub spadku nie może spowodować przejęcia przez Fundację długów przewyższających wartość mienia przyjmowanego.
+6. W sprawach przyjęcia darowizn i dziedziczenia oświadczenia wymagane przepisami prawa składa Zarząd Fundacji.
```

Comment:
chyba bardziej zrozumiałe będzie, jeśli zmienimy trochę szyk: "wymagane przepisami oświadczenia"

Reply:
@pannami tylko, o ile dobrze rozumiem, tutaj nie możemy rozdzielić `przepisami` od `prawa` więc `wymagane przepisami oświadczenia prawa` raczej nie zachowają sensu zdania.

Reply:
@OtisRed nie wiem, czy to dobrze, że odwołujemy się do `Zarządu Fundacji`, zanim go definiujemy.

Reply:
1. Ostatni komentarz w sprawie składni ZAMKNIĘTE
2. Można spróbować przerzucić te definicje do przodu dokumentu OTWARTE